util/check_translation.py

```diff
@@ -150,7 +150,7 @@ def parse_translations(triggers):
                 line = line.strip()
 
                 # remove comments (assume no comments in strings)
-                line = re.sub(r'//.*$', '', line)
+                line = re.sub(r'\s?//.*$', '', line)
```

Comment:
Should this be `\s*`?

Reply:
we can also do `\s*` but at least for all of my submissions it always matches `\s?//.*$` but will add your suggestion :3